Description: Use SSL certificates from ca-certificates package
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: not-needed
Last-Updated: 2013-05-16

--- a/lib/excon/constants.rb
+++ b/lib/excon/constants.rb
@@ -4,7 +4,7 @@
 
   CR_NL = "\r\n"
 
-  DEFAULT_CA_FILE = File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "data", "cacert.pem"))
+  DEFAULT_CA_FILE = '/etc/ssl/certs/ca-certificates.crt'
 
   DEFAULT_CHUNK_SIZE = 1048576 # 1 megabyte
 
